tools/webots.py

```diff
@@ -41,6 +42,10 @@ def register(command):
         help="Disables cleaning the build volume before compiling",
     )
 
+    build_subcommand.add_argument(
+        "-j", "--jobs", dest="jobs", action="store", help=f"Dictates the number of jobs to run in parallel ({multiprocessing.cpu_count() + 2} by default"
+    )
```

Comment:
```suggestion
    build_subcommand.add_argument(
        "-j", "--jobs", dest="jobs", action="store", default=multiprocessing.cpu_count() + 2, help=f"Dictates the number of jobs to run in parallel ({multiprocessing.cpu_count() + 2} by default"
    )
```

Reply:
We had a discussion about this at the top of the page. We left out the default value because ninja chooses it's own default, which is CPU count + 2. 
Is there another reason we should be passing it in?